### PR TITLE
Adjust project directory to conform with truffle setup

### PR DESCRIPTION
**Pull request by yyyyyyuanfei**
#99

### CODE REVIEW

**Comment by Arachnid** [comment]
Please let us know when this is ready to review.

**Comment by yuanfeiz** [comment]
@Arachnid Alright, it's ready to be reviewed. Still plenty "todos" like to use truffle for compiling sol before the tests, I'll leave them for further improvements. 

One thing I'm not sure is that this PR introduces a new dependency to [eth-ens-namehash](https://www.npmjs.com/package/eth-ens-namehash), is there a better way to ensure the canonicity over this core algorithm while avoid implementing it in multiple places?

**Comment by yuanfeiz** [comment]
And sorry I was out for three days, I'll be more responsive later.

File: migrations/2_deploy_contracts.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    });
+}
+
+/**
+ * Deploy the ENS and HashRegistrar(Simplified)
+ *
+ * @param {Object} deployer truffle deployer helper
+ * @param {string} rootNode hashed name using namehash algorithm
+ */
+function deployAuctionRegistrar(deployer, rootNode) {
```
Comment (Arachnid) [inline migrations/2_deploy_contracts.js]: This accepts an argument `rootNode`, but always attempts to register the registrar at 'eth'. It should either not take that argument (and just hardcode 'eth'), or take a label hash (eg, `sha3('eth')`) and compute the root node from that.

**Comment by Arachnid** [comment]
Thank you - this looks like a big improvement, and will also make testing the DApp easier. I'm fine with the dependency on eth-ens-namehash.

Just one outstanding comment and then I'd be happy to merge it.

File: migrations/2_deploy_contracts.js (hunk 1 above)
Comment (yuanfeiz) [inline migrations/2_deploy_contracts.js]: Good catch, fixed.

**Review by Arachnid — APPROVED** [review]

**Comment by Arachnid** [comment]
Thanks!

**Pull request merged**